CHANGELOG.md

```diff
@@ -1,27 +1,50 @@
 # Changelog
 
-## Unreleased
+## 0.2.0 - 2023-07-19
 
 ### BREAKING
 
 - **make artifact autodetect configurable - [Gankra]/[pr527]**
 
+  We now provide a new boolean key, `components.artifacts.auto`, that lets you explicitly
+  enable autodetection of artifacts. Previously, we would only enable this if you either
+  turned on `components.artifacts.cargo_dist`, or if you provided some package manager entries.
+  Since oranda _does_ also support gleaning artifacts even without `cargo-dist` support enabled,
+  we added this extra switch to let you toggle it without having to mess around with package managers.
+
+  This is a **breaking change**, as enabling `cargo-dist` support or specifying package managers does
+  not turn on auto-detection of artifacts anymore.
+
 ### Features
 
 - **Typescript syntax highlighting support - [shadows-withal]/[pr525], [geelen]/[i513]**
 
+  You can now use the `ts`/`typescript` languages in code blocks! Hooray for types!
+
 - **Better logo positioning - [SaraVieira]/[pr524], [tertsdiepraam]/[i519]**
 
+  Logos set via the `styles.logo` option will now be properly centered/aligned in all themes,
+  and set to a maximum width so that a 1920x1080 logo won't break the website completely.
+
 ### Fixes
 
 - **Hacker theme highlight color - [SaraVieira], [pr523]/[i522]** 
 
+  Selecting text in the Hacker theme now applies a nice, green, high contrast highlight background
```

Comment:
we may want to say what it previously did

Reply:
i'm not actually sure lol